source/operations/install-deploy-manage/upgrade-minio-tenant.rst

```diff
@@ -17,12 +17,126 @@ MinIO Kubernetes Operator and Plugin
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 This procedures on this page *requires* a valid installation of the MinIO Kubernetes Operator and assumes the local host has a matching installation of the MinIO Kubernetes Operator.
-This procedure assumes the latest stable Operator and Plugin version |operator-version-stable|.
+This procedure assumes the latest stable Operator and Plugin, version |operator-version-stable|.
 
 See :ref:`deploy-operator-kubernetes` for complete documentation on deploying the MinIO Operator.
 
+Install the Plugin
+++++++++++++++++++
+
 .. include:: /includes/k8s/install-minio-kubectl-plugin.rst
 
+Upgrading from Operator v4.2.2
```

Comment:
Can we add a rellink above and reference this in a step from the `upgrade-minio-operator` pages too?

Reply:
Not needed. This information is already on that page.
There's duplication involved here, but I don't see the need to clean it up in this PR.